lib/jekyll/commands/serve/servlet.rb

```diff
@@ -22,7 +22,7 @@ def initialize(server, root, callbacks)
 
         def search_file(req, res, basename)
           # /file.* > /file/index.html > /file.html
```

Comment:
Can you illustrate what this change does by modifying this comment?

`/.html > /file > /file.html`?